Elsewhen/Helpers/TimeZoneFlags.swift

```diff
@@ -486,3 +486,419 @@ let europeanUnionTimeZones: Set<String> = [
+    "America/Mazatlan" : "Mexico",
+    "America/Menominee" : "United States",
+    "America/Merida" : "Mexico",
+    "America/Metlakatla" : "United States",
+    "America/Mexico_City" : "Mexico",
+    "America/Miquelon" : "Saint Pierre and Miquelon",
+    "America/Moncton" : "Canada",
+    "America/Monterrey" : "Mexico",
+    "America/Montevideo" : "Uruguay",
+    "America/Montserrat" : "Montserrat",
+    "America/Nassau" : "Bahamas",
+    "America/New_York" : "United States",
+    "America/Nipigon" : "Canada",
+    "America/Nome" : "United States",
+    "America/Noronha" : "Brazil",
+    "America/North_Dakota/Beulah" : "United States",
+    "America/North_Dakota/Center" : "United States",
+    "America/North_Dakota/New_Salem" : "United States",
+    "America/Nuuk" : "Greenland",
+    "America/Ojinaga" : "Mexico",
+    "America/Panama" : "Panama",
+    "America/Pangnirtung" : "Canada",
+    "America/Paramaribo" : "Suriname",
+    "America/Phoenix" : "United States",
+    "America/Port-au-Prince" : "Haiti",
+    "America/Port_of_Spain" : "Trinidad and Tobago",
+    "America/Porto_Velho" : "Brazil",
+    "America/Puerto_Rico" : "Puerto Rico",
+    "America/Punta_Arenas" : "Chile",
+    "America/Rainy_River" : "Canada",
+    "America/Rankin_Inlet" : "Canada",
+    "America/Recife" : "Brazil",
+    "America/Regina" : "Canada",
+    "America/Resolute" : "Canada",
+    "America/Rio_Branco" : "Brazil",
+    "America/Santa_Isabel" : "Soloman Islands",
+    "America/Santarem" : "Brazil",
+    "America/Santiago" : "Chile",
+    "America/Santo_Domingo" : "Dominican Republic",
+    "America/Sao_Paulo" : "Brazil",
+    "America/Scoresbysund" : "Greenland",
+    "America/Shiprock" : "United States",
+    "America/Sitka" : "United States",
+    "America/St_Barthelemy" : "Saint Barthelemy",
+    "America/St_Johns" : "Canada",
+    "America/St_Kitts" : "Saint Kitts and Nevis",
+    "America/St_Lucia" : "Saint Lucia",
+    "America/St_Thomas" : "Islands",
+    "America/St_Vincent" : "Saint Vincent and the Grenadines",
+    "America/Swift_Current" : "Canada",
+    "America/Tegucigalpa" : "Honduras",
+    "America/Thule" : "Greenland",
+    "America/Thunder_Bay" : "Canada",
+    "America/Tijuana" : "Mexico",
+    "America/Toronto" : "Canada",
+    "America/Tortola" : "British Virgin Islands",
+    "America/Vancouver" : "Canada",
+    "America/Whitehorse" : "Canada",
+    "America/Winnipeg" : "Canada",
+    "America/Yakutat" : "United States",
+    "America/Yellowknife" : "Canada",
+    "Antarctica/Casey" : "Antarctica",
+    "Antarctica/Davis" : "Antarctica",
+    "Antarctica/DumontDUrville" : "Antarctica",
+    "Antarctica/Macquarie" : "Australia",
+    "Antarctica/Mawson" : "Antarctica",
+    "Antarctica/McMurdo" : "Antarctica",
+    "Antarctica/Palmer" : "Antarctica",
+    "Antarctica/Rothera" : "Antarctica",
+    "Antarctica/Syowa" : "Antarctica",
+    "Antarctica/Troll" : "Antarctica",
+    "Antarctica/Vostok" : "Antarctica",
```

Comment:
Wouldn't these already be returned by the identifier search?

Reply:
Yes, you're right, and I've removed others (like Argentina has a load, etc) - I just missed these. I'll remove them!